packages/contracts/token_contract/src/interface.sw

```diff
@@ -0,0 +1,28 @@
+library interface;
+
+abi TokenContract {
```

Comment:
```suggestion
abi Token {
```

I would omit `Contract` since it doesn't add semantic value imo.
I would also consider whether this should be renamed to `Asset`.